properties.ini.sample

```diff
@@ -0,0 +1,160 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+; hypervisor-builder can help to get some hypervisor and guest information
+
+[virtwho]
+; host= is the rhel system to run virt-who service
+; repo= is the base url of the rhel compose
+host=
+username=
+password=
+port=
+repo=
+proxy_server=
+proxy_prot=
+
+[rhsm]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+
+[satellite]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+secondary_org=
+activation_key=
+
+[vcenter]
+; esx_ datas are for the esxi host which the guest is based on
+server=
+username=
+password=
+ssh_username=
+ssh_password=
+esx_ip=
+esx_username=
+esx_password=
+esx_uuid=
+esx_hwuuid=
+esx_hostname=
+esx_version=
+esx_state=
```

Comment:
suggest to remove the hypervisor state for all the hypervisors, since there is no hypervisor state in the mapping info, only have guest state

Reply:
yeah, I just thought maybe the hypervisor state is used to estimate the environment status.

Reply:
It's true, but we should use a separate job to check the hpervisor health, it will not use this test data file